src/main/java/pyws/swyp/meeting/service/MeetingService.java

```diff
@@ -0,0 +1,40 @@
+package pyws.swyp.meeting.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import pyws.swyp.global.error.CustomException;
+import pyws.swyp.global.error.ErrorCode;
+import pyws.swyp.meeting.dto.MeetingCreateRequest;
+import pyws.swyp.meeting.entity.Meeting;
+import pyws.swyp.meeting.entity.MeetingParticipant;
+import pyws.swyp.meeting.repository.MeetingParticipantRepository;
+import pyws.swyp.meeting.repository.MeetingRepository;
+import pyws.swyp.member.entity.Member;
+import pyws.swyp.member.repository.MemberRepository;
+
+@Service
+@RequiredArgsConstructor
+public class MeetingService {
+    private final MeetingRepository meetingRepository;
+    private final MeetingParticipantRepository meetingParticipantRepository;
+    // 임시 조치
+    private final MemberRepository memberRepository;
+
+    public void createMeeting(MeetingCreateRequest request) {
+        // Todo: security 구현 되는대로 멤버 파싱 적용.
+        // 임시 조치
+        Member member = memberRepository.findById(1L).orElseThrow(
+                () -> new CustomException(ErrorCode.MEMBER_NOT_FOUND)
```

Comment:
이부분 람다 쓰기가 귀찮아서.. 제 PR에서 ErrorCode에 CustomException을 바로 반환해주는 메서드 만들어 놓긴 했는데 어떤지 말씀해주세요!
```suggestion
                ErrorCode.MEMBER_NOT_FOUND::toException
```

Reply:
오 확실히 더 깔끔하고 좋은 것 같습니다. 이렇게 수정해두도록 하겠습니다. 이거는 명근님 올려주신 거 머지 후에 현재 or 다음 PR에서 수정해두도록 하겠습니다.